.circleci/config.yml

```diff
@@ -180,6 +180,13 @@ jobs:
   'Nomodule Tests':
     executor:
       name: amphtml-large-executor
+    parameters:
+      config:
+        description: 'Which config file to use'
+        type: enum
+        enum: ['prod', 'canary']
+    environment:
+      config: << parameters.config >>
```

Comment:
Can you update the name on line 196 too?

Reply:
This isn't really necessary, it's just the name that appears _within_ the job - it already has the full title above

![image](https://user-images.githubusercontent.com/1839738/109054027-615e7e80-76ab-11eb-8cac-3de16ca50b63.png)

Reply:
Right, but when you click on a step and share a link, it's useful to know what it's running.

Reply:
Fair enough. Done + gave that one the longer name (` (using << parameters.config >>-config.json)`)

Reply:
Never mind, I think we're fine because the top-level job name does appear in the header.

https://app.circleci.com/pipelines/github/ampproject/amphtml/2539/workflows/df5bd370-c793-4be4-bba3-71fa5e318e14/jobs/28417/parallel-runs/0/steps/0-110

![image](https://user-images.githubusercontent.com/26553114/109054618-14c77300-76ac-11eb-8bd9-6369dd873ec0.png)